### ISSUE TITLE
MSFT-192[Member] - Low contrast for Learn More Button

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#52]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2.  Click on Web Extension tool WAVE for accessibility testing. 
3. Scroll down to Learn More and Click on contrast button. 
4. Now Observe the banner. 

**Expected Result**
Expected contrast ratio of 4:5:1.

**Actual Result**
Element has insufficient color contrast Ration 3:25:1 (foreground color: #FFFFFF, background color: ##278fff)

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=63cedfe6%2D52a5%2D4f33%2D9399%2D00e26b7dd4a2&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%20%2D%20Learn%20More%20Contrast
